packages/astro-uxds/css/home.css

```diff
@@ -1 +1,1014 @@
+  border-color: var(--card-color-border);
+  background: var(--color-background-surface-selected);
+  border-radius: 0 0 3px 3px;
+  align-self: stretch;
+  flex-grow: 1;
+  padding: 1.375rem;
+  padding-top: 0.625rem;
+  font-size: 1rem;
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+}
+.article-content h4,
+a.article-content h4 {
+  display: -webkit-box;
+  -webkit-line-clamp: 1;
+  overflow: hidden;
+  -webkit-box-orient: vertical;
+  margin-top: 1rem;
+  color: var(--color-text-primary);
+}
+.article-image,
+a.article-image {
+  border-radius: 4px 4px 0 0;
+  border-width: 1px 1px 0 1px;
+  border-style: solid;
+  border-color: var(--card-color-border);
+  width: 300px;
+  line-height: 0;
+}
+.article-image img,
+a.article-image img {
+  width: 100%;
+}
+.article-tags,
+a.article-tags {
+  background: var(--color-background-surface-header);
+  padding: 0 12px;
+  font-size: 0.9rem;
+  line-height: 1.5rem;
+  border-radius: 3px;
+  color: var(--primary-color);
+}
+.article-tags li,
+a.article-tags li {
+  display: inline-block;
+}
+.article-tags li:after,
+a.article-tags li:after {
+  content: " | ";
+  color: var(--primary-color);
+}
+.article-tags li:last-child:after,
+a.article-tags li:last-child:after {
+  display: none;
+}
+
+/*general*/
+.button,
+a.button {
+  border: none;
+  display: inline-block;
+  position: relative;
+  margin: 0 auto;
+  padding: 0.5rem 1rem;
+  border-radius: 3px;
+  background: var(--color-background-interactive);
+  color: var(--color-text-inverse);
+}
+
+.button:hover,
+.button:active a.button:hover,
+a.button:active {
+  text-decoration: none;
+  background: var(--color-background-interactive-hover);
+}
+
+.button-secondary,
+a.button-secondary {
+  border: 1px solid var(--color-background-interactive);
+  display: inline-block;
+  position: relative;
+  margin: 0 auto;
+  padding: 0.5rem 1rem;
+  border-radius: 3px;
+  background: transparent;
+  color: var(--color-background-interactive);
+}
+
+.button-secondary:hover,
+.button-secondary:active a.button-secondary:hover,
+a.button:active {
+  text-decoration: none;
+  background: transparent;
+  border-color: var(--color-background-interactive-hover);
+  color: var(--color-background-interactive-hover);
+}
+
+.link-arrow:after {
+  content: url("/img/_site/link-arrow.svg");
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  margin-left: 0.5em;
+  white-space: nowrap;
+}
+
+.section-container {
+  background-color: var(--color-background-surface-header);
+  padding: 2.25rem;
+  border-radius: 6px;
+}
+
+.title-block {
+  display: flex;
+  align-items: baseline;
+}
+
+a.button-secondary.title-button {
+  margin-bottom: 0.625rem;
+  margin-left: auto;
+  margin-right: 0;
+}
+
+.section-title {
+  font-size: 2.125rem;
+}
+
+.home__benefits .row {
+  gap: 3%;
+}
+
+.home__benefits .benefit {
+  background-color: var(--color-background-surface-selected);
+  border-radius: 3px;
+  padding: 20px 40px;
+  display: flex;
```

Comment:
I would add `justify-content: center`. to this class declaration as the content does not vertically center currently.